src/core/reducers/addons.js

```diff
@@ -380,9 +380,9 @@ export default function addonsReducer(
             ...state.bySlug,
             [addon.slug.toLowerCase()]: undefined,
           },
-          loadingBySlug: {
-            ...state.loadingBySlug,
-            [addon.slug.toLowerCase()]: undefined,
+          loadingByIdInURL: {
+            ...state.loadingByIdInURL,
+            [addon.slug]: undefined,
```

Comment:
Does this conflict with the other logic in this patch? I guess we don't have a `slug` to pass into this action like we do with `loadAddon`, and I guess it doesn't really matter.

Reply:
yeah, I think so. No test breaks so I guess it doesn't really matter as you said.